### PR TITLE
add auto-formatting to release and format some files

### PR DESCRIPTION
**Pull request by BitcoinUnlimitedJanitor**

### CODE REVIEW

**Comment by ptschip** [comment]
@gandrewstone something seems off here with the pointer alignment.

File: src/expedited.cpp [hunk 1]
```diff
@@ -84,7 +84,7 @@ bool CheckAndRequestExpeditedBlocks(CNode *pfrom)
     return false;
 }
 
-void HandleExpeditedRequest(CDataStream &vRecv, CNode *pfrom)
+void HandleExpeditedRequest(CDataStream& vRecv, CNode* pfrom)
```
Comment (ptschip) [inline src/expedited.cpp]: pointers are back to being left aligned?

Comment (gandrewstone) [reply src/expedited.cpp]: they should not be... I'll take a look.

**Comment by gandrewstone** [comment]
closed due to conflicts and reopened as #599 with pointer alignment fix

**Pull request closed without merging**